### ISSUE TITLE
Fix color in dark mode

### ISSUE THREAD

**Issue by agarwalsahil0210** [arc-pune/arc-front-end#131]
**Describe the bug**
- Fix the text color in pets detail page 
- Make the color of text to white in dark mode

**Expected behavior**

![Screenshot from 2022-04-05 12-14-45](https://user-images.githubusercontent.com/53162729/161695718-9db2f061-685d-4169-8a86-2be8facb4bdb.png)

**Screenshots**

![Screenshot from 2022-04-05 12-14-34](https://user-images.githubusercontent.com/53162729/161695752-04e171a8-cdf5-4afb-905b-f52bc4416d9b.png)

**Comment by timagixe** [comment]
Title `Word from our Founder` is not visible in dark mode.

<img width="1813" alt="Screen Shot 2022-04-05 at 5 58 48 PM" src="https://user-images.githubusercontent.com/60817786/161784025-dc691b11-7c6d-4afe-8f33-6228704da53a.png">

**Comment by agarwalsahil0210** [comment]
> Title `Word from our Founder` is not visible in dark mode.
> 
> <img alt="Screen Shot 2022-04-05 at 5 58 48 PM" width="1813" src="https://user-images.githubusercontent.com/60817786/161784025-dc691b11-7c6d-4afe-8f33-6228704da53a.png">

yeah that also seems to be an issue. Did you fix both of them @timagixe ?

**Comment by timagixe** [comment]
@agarwalsahil0210
https://github.com/arc-pune/arc-front-end/pull/133#issuecomment-1089038305

**Comment by hs-07** [comment]
hey buddy @agarwalsahil0210 i would love to work over this project so please assign me this task.
